app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list) -> int:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(players: list) -> None:
+    for player in players:
+        if issubclass(player.__class__, Elf):
```

Comment:
There is no need to call for dunder method here, in case you would like to check the type use the following syntax instead
```suggestion
        if isinstance(player, Elf):
```